### PR TITLE
Added TrustProxies Middleware for Reverse Proxy Compatability

### PR DESCRIPTION
**Pull request by nichtmonti**
"When running your applications behind a load balancer that terminates TLS / SSL certificates, you may notice your application sometimes does not generate HTTPS links." - https://laravel.com/docs/5.6/requests#configuring-trusted-proxies

This can be avoided using the trusted proxies introduced in Laravel 5.5. For some reason the changes were not included in Attendize, i assume their were missed when updating Laravel.

The commit brings the trusted proxy middleware up to date with respect to the Laravel 5.8 master branch.

### CODE REVIEW

**Comment by etiennemarais** [comment]
@nichtmonti Thank you for your contribution here. We'd like to get this into `next` soon. Please can you have a look at the conflicts before I am able to merge this? Please pull `next` branch into yours and you will be able to see the conflict there 🙏

**Comment by CristianDeluxe** [comment]
@etiennemarais I can confirm that all these changes have already been made by merging my PR update Laravel (Except the custom IPs that I do not think relevant to include)

One of the reasons I did the Laravel update from scratch is because many changes between versions were missing.

Now it's a new Laravel 6 project with Attendize code included, so all these bugs should be fixed.

@nichtmonti Could you check, cloning the next branch, if it works for you?

**Comment by nichtmonti** [comment]
@etiennemarais Merge conflicts are fixed. Seems to be mostly Laravel 6 related stuff. 
My original pull request was just to quickly add the reverse proxy functionality to the current 5.x version, if this gets updated to Laravel 6 with the next merge anyway this PR can be closed. 

@etiennemarais I haven't worked with Laravel 6.x yet and am not familiar with the changes introduced in the new major version so I cannot really comment on this.

File: composer.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
         "php-http/curl-client": "^1.7",
         "php-http/message": "^1.6",
         "predis/predis": "~1.1",
+        "stripe/stripe-php": "^6.43",
```
Comment (etiennemarais) [inline composer.json]: Please remove this part? It's breaking versioning for the later version of `"stripe/stripe-php": "^7.1",`

Comment (nichtmonti) [reply composer.json]: sorry, is resolved now

**Pull request merged**